DICOM/src/Microsoft.Health.DeID.SharedLib/EncryptFunction.cs

```diff
@@ -7,16 +7,29 @@
 using System.IO;
 using System.Security.Cryptography;
 using System.Text;
-using Newtonsoft.Json;
+using Microsoft.Health.DeID.SharedLib.Settings;
 
 namespace Microsoft.Health.Dicom.DeID.SharedLib
 {
     public class EncryptFunction
     {
         // AES Initialization Vector length is 16 bytes
         private const int AesIvSize = 16;
+        private readonly byte[] _aesKey;
+        private readonly byte[] _privateKey;
+        private readonly byte[] _publicKey;
 
-        public static byte[] EncryptContentWithAES(string plainText, byte[] key, Encoding encoding = null)
+        public EncryptFunction(EncryptSetting encryptionSetting = null)
+        {
+            encryptionSetting ??= new EncryptSetting();
+            encryptionSetting.Validate();
+
+            _aesKey = Encoding.UTF8.GetBytes(encryptionSetting.EncryptKey ?? Guid.NewGuid().ToString("N"));
```

Comment:
Similar to the comment in EncryptSetting class, use UTF8 to get bytes from a string key is unreasonable.